install_config/revhistory_install_config.adoc

```diff
@@ -14,24 +14,29 @@
+|Added NetworkManager to the
+link:../install_config/install/prerequisites.html#system-requirements[System Requirements]
+section for nodes.
+|Added
+link:../install_config/install/prerequisites.html#prereq-networkmanager[NetworkManager]
+as a prerequisite in the
+link:../install_config/install/prerequisites.html#envirornment-requirements[Environment
+Requirements] section.
 
-|xref:../install_config/configuring_authentication.adoc#[Configuring Authentication]
-|Added a note on how to obtain the xref:../install_config/configuring_authentication.adoc#HTPasswdPasswordIdentityProvider[`htpasswd`] utility.
+|link:../install_config/install/advanced_install.html[Installing -> Advanced Installation]
+|Replaced the `*openshift_docker_log_options*` Ansible variable with `*openshift_docker_options*` in the link:../install_config/install/advanced_install.html#configuring-host-variables[Configuring Host Variables] section.
 
-|link:../install_config/install/prerequisites.html[Installing -> Prerequisites]
-|Added NetworkManager as a system requirement for nodes.
+|link:../install_config/install/docker_registry.html[Installing -> Deploying a Docker Registry]
+|Fixed examples in the link:../install_config/install/docker_registry.html#securing-the-registry[Securing the Registry] section to use consistent `--cert` and `--key` values. Also, clarify the origin of the *_ca.crt_* file that must be installed per-node.
 
-|xref:../install_config/install/docker_registry.adoc#[Installing -> Deploying a Docker Registry]
-|Fixed examples in the xref:../install_config/install/docker_registry.adoc#securing-the-registry[Securing the Registry] section to use consistent `--cert` and `--key` values. Also, clarify the origin of the *_ca.crt_* file that must be installed per-node.
+|link:../install_config/configuring_authentication.html[Configuring Authentication]
+|Added a note on how to obtain the link:../install_config/configuring_authentication.html#HTPasswdPasswordIdentityProvider[`htpasswd`] utility.
 
-|xref:../install_config/web_console_customization.adoc#[Customizing the Web Console]
-|Added that each time a user's token to {product-title} expires, the user is presented with a custom page. Also, added xref:../install_config/web_console_customization.adoc#custom-login-page-example-usage[use cases] for custom login pages.
-|xref:../install_config/install/advanced_install.adoc#configuring-host-variables[Installing -> Advanced Installation]
+|link:../install_config/web_console_customization.html[Customizing the Web Console]
+|Added that each time a user's token to {product-title} expires, the user is presented with a custom page. Also, added link:../install_config/web_console_customization.html#custom-login-page-example-usage[use cases] for custom login pages.
+|link:../install_config/install/advanced_install.html#configuring-host-variables[Installing -> Advanced Installation]
 |Updated `*openshift_router_selector*` to its new name of `*openshift_hosted_router_selector*`.
```

Comment:
Shoot, I missed combining this w/ the other Advanced Installation. Oh well, next time.